content/developer/blueprints/spec-deployment-settings.md

```diff
@@ -0,0 +1,146 @@
+layout: bt_wiki
+title: Deployment Settings
+category: Blueprints
+draft: false
+weight: 1000
+aliases: /blueprints/spec-deployment-settings/
+---
+
+Deployment Settings are additional settings and instructions provided to 
+Cloudify manager on how to handle a deployment created out of this blueprint.
+
+# Declaration
+
+## Default Schedules setting
+
+This deployment setting allows you to define a list of deployment schedules 
+to be created for any deployment which is based on this blueprint.
+
+{{< highlight  yaml >}}
+deployment_settings:
+  default_schedules:
+    sc-1:
+      workflow: ...
+      since: ...
+      timezone: ...
+      recurring: ...
+      count: ...
+    sc-2:
+      ...
+{{< /highlight >}}
+
+
+## Default Schedules schema
+
+Keyname               | Required | Type        | Description
+-----------           | -------- | ----        | -----------
+`workflow`            | yes      | string      | The name of the workflow the schedule should execute, e.g. `install`.
+`execution_arguments` | no       | dict        | Schema of arguments to be passed directly to the workflow execution (detailed below).
+`workflow_parameters` | no       | dict        | Schema of additional parameters to be passed to the workflow execution as kwargs.
+`since`               | yes      | string      | A string representing the earliest possible time for the execution to run. Supported formats: `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+`until`               | no       | string      | A string representing the latest possible time for the execution to run. Supported formats: `YYYY-MM-DD HH:MM`, `HH:MM`, or a relative time expression such as `+2 weeks` or `+1day+10min`.
+`recurring`           | no       | string      | Recurrence on the scheduled execution. e.g. `2 weeks`, `30 min` or `1d`. Mandatory if `count` is other than 1.
+`timezone`            | no       | string      | The timezone to be used for scheduling, e.g. `EST` or `Asia/Jerusalem`. By default, the local timezone will be used. Supports any timezone in the [tz database](en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+`weekdays`            | no       | list        | A list of weekdays on which to run the execution, e.g. `su,mo,tu`. If left empty, runs on all weekdays.
+`count`               | no       | number      | Maximum number of times to run the execution. If left empty, there's no limit on repetition.
+`default_enabled`     | no       | boolean     | If set to `false`, the created schedule will not run executions until manually enabled by the user. Default: `true`.
+`stop_after_fail`     | no       | boolean     | If set to `true`, once the execution has failed, the scheduler won't make further attempts to run it. Default: `false`.
+`slip`                | no       | number      | Maximum time window after the target time has passed, in which the scheduled execution can run (in minutes). If not provided, default: 0.
```

Comment:
In `default_enabled` and `stop_after_fail`, the description doesn't say "If not provided" before specifying the default. What should be the right way?